encord/project.py

```diff
@@ -121,6 +121,13 @@ def datasets(self) -> list:
         """
         return self._project_instance.datasets
 
+    @property
+    def project_type(self) -> str:
```

Comment:
I'd suggest to return the ProjectType enum from here rather than a raw string

Reply:
Changed to ProjectType